### DISCUSSION TITLE
String exceeds max length but len(text) is under limit

### DISCUSSION THREAD

**Discussion by scoooobert** [milvus-io/milvus#36708] [Q&A and General discussion] [answered]
I am trying to insert data into milvus in text field is varchar(60000), I insert text which I check len(text) = 21033 which under limit but in milvus show ```MilvusException: (code=1100, message=the length (66976) of 0th string exceeds max length (60000): invalid parameter[expected=valid length string][actual=string length exceeds max length]```

**Comment by xiaofan-luan** [comment, marked as answer]
> I am trying to insert data into milvus in text field is varchar(60000), I insert text which I check len(text) = 21033 which under limit but in milvus show `MilvusException: (code=1100, message=the length (66976) of 0th string exceeds max length (60000): invalid parameter[expected=valid length string][actual=string length exceeds max length]`

The size of varchar is actually by byte , but not by token (For example, unicode may take 2-4 byte per token)
